pkg/service/model_build_listener.go

```diff
@@ -19,12 +19,16 @@ func (t *defaultModelBuildTask) buildListeners(ctx context.Context, scheme elbv2
 		return err
 	}
 
-	for _, port := range t.service.Spec.Ports {
-		_, err := t.buildListener(ctx, port, *cfg, scheme)
-		if err != nil {
-			return err
+	for _, service := range t.stack.ListServices() {
+		t.service = &service
```

Comment:
This will leave `t.service` as the last service in the listed services. Is that intentional? Seems like a code-smell.

Reply:
Yeah, it would really matter since it is not used anywhere else, but I added back to assign to previous service just in case.